src/client/services/board/persistence/persistence.js

```diff
@@ -49,15 +48,14 @@ var PersistenceService = SparkBase.extend({
    * Uploads image to spark files and adds SCR + downalodUrl to the persistence
    * service
    * @memberof Board.PersistenceService
-   * @param  {Conversation} conversation - Contains the currently selected conversation
    * @param  {Board~Channel} channel
    * @param  {File} image - image to be uploaded
    * @return {Promise<Board~Content>}
    */
-  addImage: function addImage(conversation, channel, image) {
-    return this.spark.board._uploadImage(conversation, image)
+  addImage: function addImage(channel, image) {
+    return this.spark.board._uploadImage(channel, image)
       .then(function addContent(scr) {
-        return this.spark.board.persistence.addContent(conversation, channel, [{
+        return this.spark.board.persistence.addContent(channel, [{
```

Comment:
Can we just do:
.then((scr) => this.spark.board.persistence.addContent(channel .......)